Viper Practice/src/presenter/FruitesPresenter.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  FruitesPresenter.swift
+//  Viper Practice
+//
+//  Created by Huzaifa Umer on 20/12/2023.
+//
+
+import Foundation
+
+protocol FruitesPresenterProtocol : AnyObject {
+    func loadView()
+    func fruitesFetched(_ fruits: [Fruit])
+    func getFruites() -> [Fruit]
+}
+
+class FruitesPresenter : FruitesPresenterProtocol {
+    
+    weak var view : FruitesViewProtocol?
+    var interactor: FruitesInteractorProtocol?
+    var router: FruitesRouterProtocol?
+    
+    private var fruites = [Fruit]()
+    
+    func loadView() {
```

Comment:
can be renamed to:
loadData()
fetchFruites()

Reply:
fixed